### ISSUE TITLE
Studio doesn't show index

### ISSUE THREAD

**Issue by JohnyL** [Microsoft/sqlopsstudio#215]
- SQL Operations Studio Version: sqlops 0.23.6 (8a3011756318a2ded1e6e84332230a2b2bf9c544, 2017-11-06T21:51:43.232Z)
- OS Version: Windows_NT x64 10.0.16299
---

A table in the database has primary key (non-clustered) and clustered index. However, the editor doesn't show primary key. Moreover, it would be welcome to have distinct icons for keys and indexes like in SSMS.

**Comment by erickangMSFT** [comment]
@JohnyL Can you please elaborate 'the editor doesn't show primary key' part? I am trying to understand this issue better with more information :)

**Comment by JohnyL** [comment]
@erickangMSFT A picture is worth a thousand words :)

![SQL OPS Editor Bug](https://i.imgur.com/C1IR6qX.png)

**Comment by kevcunnane** [comment]
I did some digging and this looks like a bug we ported over from SSDT (which we based our Object Explorer implementation on since it's cleaner than SSMS). In SSDT, foreign keys show as entries under `Keys` and `Indexes`, but PKs only show under `Keys`. In SSMS they show in both.

Given FKs show in both it seems like this must be a bug where PKs don't show rather than an intentional difference. @llali we can update the XML file to get this working?
